translations/README-cn.md

```diff
@@ -188,31 +188,33 @@
 
 在学习过程中，我使用 GitHub 特殊语法的 markdown 去检查计划的进展，包括使用包含任务进度的任务列表。
 
-**创建一个新的分支，以便你可以像这样去勾选计划的进展：直接往方括号中填写一个字符 x 即可：[x]**。
+创建一个新的分支，以便你可以像这样去勾选计划的进展：直接往方括号中填写一个字符 x 即可：[x]。
 
-    Fork一个分支，并跟随以下的指令
+```sh
+Fork 一个分支，并跟随以下的指令
+```
 
 通过单击 Fork 按钮来 fork GitHub 仓库：[jwasham/coding-interview-university](https://github.com/jwasham/coding-interview-university)
 
-    克隆项目到本地
-
-`git checkout -b progress`
-
-`git remote add jwasham https://github.com/jwasham/coding-interview-university`
-
-`git fetch --all`
-
-    在你完成了一些修改后，在框框中打 x
-
-`git add .`
-
-`git commit -m "Marked x"`
-
-`git rebase jwasham/main`
-
-`git push --set-upstream origin progress`
-
-`git push --force`
+克隆项目到本地
+
+```sh
+git clone git@github.com:<your_github_username>/coding-interview-university.git
```

Comment:
this needs to clone this project. Remove <your_github_username>

Reply:
I see the English version is the same. Should I change it like this

```suggestion
git clone git@github.com:/coding-interview-university.git
```

Reply:
You're right, it is the same on the English version. I don't think that works, though.

Reply:
So should we change the English version first, without reference, I don't know how to modify, this is just the translation.

Reply:
Thank you! I've updated it: https://github.com/jwasham/coding-interview-university/commit/7c6679f7ac0801ef5487480f1a8cc5dfaea84453

The user has to fork to make their own repo, then clone that.